app/src/main/java/com/example/keybindhelper/Adapters/ProjectAdapter.kt

```diff
@@ -39,7 +39,7 @@ class ProjectAdapter(
         val p = projectList[position]
         (holder.itemView.findViewById<View>(R.id.project_name) as TextView).text = p.name
         val cal = Calendar.getInstance()
-        cal.time = p.lastAccessed
+        cal.time = p.lastAccessed!!
```

Comment:
Avoid force unwrap (!!) at all costs.  That can create a NullPointerException, if the variable value is null.
Use something null safe, like foo?.let{} ?: instead.